eip.tf

```diff
@@ -1,6 +1,41 @@
+locals {
+  # if eip map not defined at all, want_eip is false.
```

Comment:
I think this and the next comment should be merged & made clearer.

Reply:
Probably so, and perhaps the documentation suffices that we don't need these comments?